src/components/SortableTable/SortableTable.tsx

```diff
@@ -45,19 +65,41 @@ const SortableTable: React.FC<SortableTableProps> = ({ data, columns }) => {
   const getSortIcon = (key: string) => {
     const { sortKey, sortOrder } = sortState;
     if (sortKey === key) {
-      return sortOrder === "asc" ? "↕︎" : "↕︎";
+      return sortOrder === "asc" ? "↓" : "↑";
     }
-    return "";
+    return "↑↓";
   };
 
   const sortedData = useMemo(() => {
     const { sortKey, sortOrder } = sortState;
+    const isArrayOfStrings = (content: any): content is HtmlString[] => {
+      return (
+        Array.isArray(content) &&
+        content.every((item) => typeof item === "string")
+      );
+    };
+    const getValue = (content: Stats | number | string | HtmlString[]) => {
+      if (typeof content === "object" && content !== null) {
+        if ("mean" in content) {
+          return content.mean;
+        }
+      }
+      return content;
+    };
     if (sortKey) {
       return [...data].sort((a, b) => {
-        if (a[sortKey] < b[sortKey]) {
+        const valueA = getValue(a[sortKey]);
+        const valueB = getValue(b[sortKey]);
+
+        // Exclude HtmlString from sorting
+        if (isArrayOfStrings(valueA) || isArrayOfStrings(valueB)) {
```

Comment:
Instead of looking at a cell value you could also specify in the Header if you want it sortable and if sortable what kind of sort.

Something like

```js
const headers = [{
key: 'key1',
value: ''Label 1',
// By default sort 
}, {
key: 'key2',
value: ''Label 2',
sort: false,  // do not sort
}, {
key: 'key3',
value: ''Label 3',
sort: 'mean', // if string use as key of cell object
}]
return <SortableTable verticalHeaders={headers} .../>
``

Reply:
Fixed!